build.sh

```diff
@@ -21,7 +29,9 @@ cmake -DCMAKE_BUILD_TYPE=Release \
   -GNinja \
   -DCMAKE_C_COMPILER=$CC \
   -DCMAKE_CXX_COMPILER=$CXX \
-  -DDEPOT_TOOLS_DIR="$root/depot_tools" \
+  -DDEPOT_TOOLS_DIR=$DEPOT_TOOLS_DIR \
   "$@"
 
+ccache -s # print the stats after the build
```

Comment:
Should we print this after the main build?
```
2021-11-01T02:41:24.9858097Z cache size                          18.2 MB
2021-11-01T02:41:24.9858632Z max cache size                     200.0 MB
```

Reply:
oh, right 🤦 i'll change

Reply:
done